.github/workflows/optimize-backend-linting.yml

```diff
@@ -2,14 +2,20 @@ name: Optimize Backend Linting
 on:
   push:
     branches:
-      - master
+      - main
+      - maintenance/**
+      - release/**
+    paths:
```

Comment:
There's just this git error saying that it's not possible to use these 2 directives at the same time.
I think a correct syntax for this would be:
```
paths:
  - "optimize/**"
  - ".github/workflows/optimize-code-scanning-java.yml"
  - "!optimize/client/**"